hooks/shotgun_filters.py

```diff
@@ -0,0 +1,222 @@
+# Copyright (c) 2020 Autdesk, Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Autodesk, Inc.
+import sgtk
+import pprint
+
+HookBaseClass = sgtk.get_hook_baseclass()
+
+
+class ShotgunFilters(HookBaseClass):
+    """
+    Controls the filter configuration for the Shotgun Panel.
+
+    Via this hook, the data that is retrieved for the Shotgun Panel can be controlled.
+    """
+
+    def get_link_filters(self, sg_location, entity_type, context_project, context_user):
```

Comment:
I'm guessing this might be old logic from Manne that's being pushed into a hook now. If that's true, then apologies for asking you to clean up a couple things below.

Reply:
Yeah, I copied this `get_link_filters` method from the dialog to expose it in the hook. No problem though, happy to improve things while we're here anyway.